pod-resizer/README.md

```diff
@@ -0,0 +1,28 @@
+# pod-resizer
+
+This container image watches over another container, and vertically scales the
```

Comment:
watches over another container *in a deployment* (that part is pretty important to mention)